python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -8,12 +8,16 @@
 
 import numpy as np
 import pandas as pd
+import torch
```

Comment:
please put these below in the try/catch.  pytorch is NOT required for this package, since we want this package to be used by users even if they don't have pytorch installed.  If you don't move this to the try/catch, users without pytorch will see import of ml_wrappers package error out since they don't have pytorch.